### DISCUSSION TITLE
SelectionboxWidget

### DISCUSSION THREAD

**Discussion by snehabihani** [Blazor-Diagrams/Blazor.Diagrams#238] [Q&A]
Can we changes selectionbox visibility from shiftkey to ctrlkey? or instead selecting nodes with ctrlkey  I need with shift key? so selection keys will be same?

**Comment by snehabihani** [comment]
Please any solution for this.

**Reply by zHaytam** [reply]
Hello, sorry but right now the shortcut is not configurable so it's not really possible.
What you can do is take the code for the SelectionBoxWidget and modify it to your needs for now.

**Reply by snehabihani** [reply]
ohk Thank you.. Can we add PR for this? we can add enum for ctrl and shift and user can opt through diagram options

**Reply by snehabihani** [reply]
Is this will be useful? Or else we can change shift to cntrl so all selections will be with same key
